### DISCUSSION TITLE
Dark Theme

### DISCUSSION THREAD

**Discussion by barepixels** [MoonRide303/Fooocus-MRE#69] [General]
Fooocus have a dark mode already. No need to make a new theme. Only need to have the option to set default auto-launch in dark mode in "settings.json"

To view Fooocus dark mode, change your URL to "http://127.0.0.1:7860/?__theme=dark"

![dark](https://github.com/MoonRide303/Fooocus-MRE/assets/92839033/32149682-2da4-4d6f-b2fa-be88734e46a1)

**Comment by ppricha75** [comment]
It's working good, but how to have this permanently ?
Thanks

**Comment by mikejarxyz** [comment]
yes, where is the settings file you speak of?

**Reply by ppricha75** [reply]
Hi,
I'm seeking the dark theme mode for Fooocus MRE. It's working with your hint ("http://127.0.0.1:7860/?__theme=dark") but how to make this command permanent ? I tried in the batch file, but I've got an error message.
Thank's for your time.
Pascal
